HomeTask5/Task2/Task5_2_TCP_Client.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+	conn, err := net.Dial("tcp", "127.0.0.1:8080")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for {
+		reader := bufio.NewReader(os.Stdin)
+
+		fmt.Print("Text to send: ")
+
+		text, err := reader.ReadString('\n')
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		fmt.Fprintf(conn, text+"\n")
```

Comment:
`Fprintf` func means **print f**ormatted , so it can be used as `fmt.Fprintf(conn, "%s\n", text)` or just use `fmt.Fprint`